google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/prepare/datacatalog_strings_helper.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import six
+
+
+class DataCatalogStringsHelper:
+    __UTF8_CHARACTER_ENCODING = 'UTF-8'
+    # String field values are limited to 2000 bytes size when encoded in UTF-8.
+    __STRING_VALUE_UTF8_MAX_LENGTH = 2000
```

Comment:
This is the max length for Tag Fields. Since the refactored class is intended to serve multiple purposes, I suggest always asking for the max length to be considered, which means no constant value here and no default value in the `truncate_string` method. WDYT? 🤔